qemu/tests/cfg/numa_opts.cfg

```diff
@@ -1,67 +1,74 @@
 - numa_opts:
-  type = numa_opts
-  # paused VMs are enough for our purposes, no need to boot them:
-  paused_after_start_vm = yes
-  kill_vm = yes
-  kill_vm_gracefully = no
-  encode_video_files = no
-  pre_command = "sync && echo 3 >/proc/sys/vm/drop_caches;"
-  smp = 6
-  mem = 4096
-  vcpu_maxcpus = ${smp}
-  backend_mem = memory-backend-ram
-  use_mem = no
-  variants:
-    - nodes.0:
-      # no extra parameters => zero nodes
-      numa_nodes = 0
-      # on ppc,no extra parameters => one node with size = mem
-      ppc64le, ppc64:
-          numa_nodes = 1
-    - nodes.1:
-      numa_nodes = 1
-      mem_devs = "mem0"
-      size_mem0 = 4096M
-      guest_numa_nodes = "node0"
-      numa_memdev_node0 = mem-mem0
-      numa_cpus_node0 = "0,1,2,3,4,5"
-    - nodes.2:
-      numa_nodes = 2
-      mem_devs = "mem0 mem1"
-      size_mem0 = 1024M
-      size_mem1 = 3072M
-      guest_numa_nodes = "node0 node1"
-      numa_memdev_node0 = mem-mem0
-      numa_memdev_node1 = mem-mem1
-      numa_cpus_node0 = "4,5"
-      numa_cpus_node1 = "0,1,2,3"
-    - nodes.3:
-      numa_nodes = 3
-      mem_devs = "mem0 mem1 mem2"
-      size_mem0 = 512M
-      size_mem1 = 1024M
-      size_mem2 = 2560M
-      guest_numa_nodes = "node0 node1 node2"
-      numa_memdev_node0 = mem-mem0
-      numa_memdev_node1 = mem-mem1
-      numa_memdev_node2 = mem-mem2
-      numa_cpus_node0 = "0,1"
-      numa_cpus_node1 = "2,3"
-      numa_cpus_node2 = "4,5"
-    - nodes.128:
-        type = numa_maxnodes
-        numa_nodes = 128
-        mem_fixed = 4G
-        vm_mem_minimum = 4G
-        node_size = 32M
-        start_vm = no
-        ppc64,ppc64le:
-            mem_fixed = 32G
-            node_size = 256M
-            vm_mem_minimum = 32G
-        paused_after_start_vm = no
-        prealloc_mem = yes
-        policy_mem = default
-        Linux:
-            numa_cmd = "cat /sys/devices/system/node/possible"
-            numa_expected = "0-127"
+    type = numa_opts
+    kill_vm = yes
+    kill_vm_gracefully = no
+    encode_video_files = no
+    pre_command = "sync && echo 3 >/proc/sys/vm/drop_caches;"
+    smp = 6
+    mem = 4096
+    vcpu_maxcpus = ${smp}
+    backend_mem = memory-backend-ram
+    use_mem = no
+    variants:
+      - nodes.0:
+        # no extra parameters => one node in guest os
+        guest_expect_nodes = 1
+        # no extra parameters => zero nodes with qemu monitor
+        monitor_expect_nodes = 0
+        # on ppc,no extra parameters => one node and size = mem with qemu monitor
+        ppc64le, ppc64:
+            monitor_expect_nodes = 1
+        variants:
+            - @default:
+            - with_maxmem:
```

Comment:
Hi, I just noticed this case only works for q35, please add 'only q35', thanks!

Reply:
> Hi, I just noticed this case only works for q35, please add 'only q35', thanks!

Updated, thanks ~